### PR TITLE
packaging: make branch configurable

### PR DESCRIPTION
**Pull request by MaximilianMeister**
Signed-off-by: Maximilian Meister <mmeister@suse.de>

### NOTE

This is related to our packaging automation to support stable branch packaging

### DEPENDS ON

https://gitlab.suse.de/MaximilianMeister/automation/commit/6b686a21797515813963a397927e3811ef7f386b within https://gitlab.suse.de/docker/automation/merge_requests/148

### CODE REVIEW

**Comment by jordimassaguerpla** [comment]
lgtm

File: packaging/suse/make_spec.sh [hunk 1]
```diff
@@ -45,22 +46,22 @@ Summary:        Production-Grade Container Scheduling and Management
 License:        Apache-2.0
 Group:          System/Management
 Url:            http://kubernetes.io
-Source:         master.tar.gz
+Source:         ${BRANCH}.tar.gz
```
Comment (kiall) [inline packaging/suse/make_spec.sh]: This assumes a branch will never have a / in it (e.g. feature/foo)? 

    BRANCH=${2:-master}
    SAFE_BRANCH=${BRANCH//\//-}

should make it more reliable?

Comment (MaximilianMeister) [inline packaging/suse/make_spec.sh]: good point! will fix

**Comment by MaximilianMeister** [comment]
[EPIC: Migrate infrastructure to stable release projects](https://trello.com/c/LFrQYvHY/283-epic-migrate-infrastructure-to-stable-release-projects)

**Pull request merged**